README.md

```diff
@@ -1 +1,34 @@
-orphan readme
+# HollowBot
+
+<img src="/hollowbotalpha.png" alt="HollowBot" width="256px" />
+
+[![Build Status](https://img.shields.io/travis/hollowverse/hollowbot.svg)](https://travis-ci.org/hollowverse/hollowbot)
+[![Greenkeeper badge](https://badges.greenkeeper.io/hollowverse/hollowbot.svg)](https://greenkeeper.io/hollowverse/hollowbot)
+[![Discord Badge](https://img.shields.io/discord/308394001789353985.svg)](https://discordapp.com/invite/KmnPYnu)
+
+
+## Information
+
+HollowBot is our custom ChatOps solution and it functions as a control interface to our [Release Manager](https://github.com/hollowverse/release-manager). It also provides access to critical information like our latest build status or other necessary metrics. HollowBot lives on our Discord server.
+
+### Getting Started
+
+### Prerequisites
+
+* Install latest stable [Node.js](https://nodejs.org/en/)
+* Install [Yarn](https://yarnpkg.com/lang/en/docs/install/) globally.
+* Install dependencies: `yarn install` in this directory
+
+### Development
+
+* Run `yarn activate-dev` to run HollowBot on development mode via `nodemon`.
+
+* Run `yarn activate` to run HollowBot on production mode.
+
+### Current Functionality
+
+* Type `!commands` to get the full list of commands available.
```

Comment:
I agree. There's not enough functionality to justify a Wiki yet so I just left the main `!commands` on the Readme.